var/lib/openhab2/jsondb/org.eclipse.smarthome.core.items.Item.json

```diff
@@ -115,17 +105,14 @@
       "category": "switch"
     }
   },
-  "TempSetpoint": {
+  "MyHumiProxy": {
```

Comment:
Where are these labels visible in openhab?

Reply:
In PaperUI you can see all the Items in Configuration -> Items. There is a search function. 

Personally, I tend to use the REST API Docs, Karaf Console, or look in the JSONDB directly. The problem with this check in is apparently it decided to reorder the entries. 😠 I think this is because I removed some Items. I've only seen it do this much reordering one other time. So many of the changes it is pointing out are false changes caused by the reordering.